src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -44,12 +44,10 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 		// Check to see if any of the MIME types have a renderer associated with
 		// them. If they do, prefer the renderer.
 		for (const mimeType of Object.keys(output.data)) {
-			if (mimeType === 'text/plain') {
-				continue;
-			}
-
-			// Don't render HTML outputs here; we'll render them as raw HTML below
-			if (mimeType === 'text/html') {
+			// Don't use a renderer for non-widget MIME types
+			if (mimeType === 'text/plain' ||
+				mimeType === 'text/html' ||
+				mimeType === 'image/png') {
```

Comment:
I wonder if it would be cleaner (only if this will ever expand to more mimes) to do `['text/plain', 'text/html', 'image/png'].includes(mimeType)`? 
If the array is predefined (and v8 may even do this automatically for you) I don't think there would be much performance hit at all.

Reply:
I don't personally mind either way but have seen `||` used more frequently throughout our code than `.includes`.